src/python/benchUtil.py

```diff
@@ -1488,12 +1488,15 @@ def getAntClassPath(checkout):
 
   # We use the jar file for core to leverage the MR JAR
   core_jar_file = None
+  jars_found = 0
   for filename in os.listdir('%s/lucene/build/core' % path):
     if reCoreJar.match(filename) is not None:
       core_jar_file = '%s/lucene/build/core/%s' % (path, filename)
-      break
+      jars_found += 1
   if core_jar_file is None:
     raise RuntimeError('can\'t find core JAR file in %s' % ('%s/lucene/build/core' % path))
+  if jars_found > 1:
+    raise RuntimeError('Found %d jars in %s, please only include 1 jar file' % (jars_found,'%s/lucene/build/core' % path))
```

Comment:
Maybe change message to `Found %d jars in "%s"; please run "gradle clean" to clean up` or so?

Reply:
Actually just realized in this code path the `build.gradle` file wouldn't exist (see other comment). So then wouldn't `gradle clean` not work?

Reply:
OH, yes, then suggest `ant clean` here and `gradle clean` in the other code path?  Or vice/versa :)